layouts/partials/head.html

```diff
@@ -21,6 +21,12 @@
     title="RSS Feed for dev.maxmind.com"
   />
 
+  {{ $sidebar := resources.Get "js/sidebar.ts" | js.Build }}
+  {{ $js := slice $sidebar | resources.Concat "js/bundle.js" }}
+  <script src="{{ $js.Permalink }}" defer></script>
+
+  <script defer language="javascript" type="text/javascript"  src="{{ "js/sidebar.js" | urlize | relURL }}"></script>
```

Comment:
We can remove this if we define the sidebar/menu structure in hugo.  Also, the second `<script>` tag is not needed since it's being bundled